[2021-12-23 16:01:57,054] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: test.make_indicator 2021-12-23T15:55:00+00:00 [queued]>
[2021-12-23 16:01:57,066] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: test.make_indicator 2021-12-23T15:55:00+00:00 [queued]>
[2021-12-23 16:01:57,066] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-12-23 16:01:57,067] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-12-23 16:01:57,067] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-12-23 16:01:57,086] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): make_indicator> on 2021-12-23T15:55:00+00:00
[2021-12-23 16:01:57,087] {standard_task_runner.py:52} INFO - Started process 336431 to run task
[2021-12-23 16:01:57,093] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'test', 'make_indicator', '2021-12-23T15:55:00+00:00', '--job-id', '1740', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpnzg83rmp', '--error-file', '/tmp/tmp8l2no2rn']
[2021-12-23 16:01:57,094] {standard_task_runner.py:77} INFO - Job 1740: Subtask make_indicator
[2021-12-23 16:01:57,135] {logging_mixin.py:104} INFO - Running <TaskInstance: test.make_indicator 2021-12-23T15:55:00+00:00 [running]> on host airflowvm
[2021-12-23 16:01:57,240] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=test
AIRFLOW_CTX_TASK_ID=make_indicator
AIRFLOW_CTX_EXECUTION_DATE=2021-12-23T15:55:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-23T15:55:00+00:00
[2021-12-23 16:01:57,241] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-12-23 16:01:57,241] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'echo 211228']
[2021-12-23 16:01:57,246] {subprocess.py:75} INFO - Output:
[2021-12-23 16:01:57,247] {subprocess.py:79} INFO - 211228
[2021-12-23 16:01:57,248] {subprocess.py:83} INFO - Command exited with return code 0
[2021-12-23 16:01:57,266] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=test, task_id=make_indicator, execution_date=20211223T155500, start_date=20211223T160157, end_date=20211223T160157
[2021-12-23 16:01:57,287] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-12-23 16:01:57,307] {local_task_job.py:151} INFO - Task exited with return code 0
